corehq/apps/app_manager/views.py

```diff
@@ -961,6 +962,13 @@ def view_generic(req, domain, app_id=None, module_id=None, form_id=None, is_user
         context.update({
             'case_properties': get_all_case_properties(app),
         })
+
+        if toggles.FORM_LINK_WORKFLOW.enabled(req.user.username):
+            modules = filter(lambda m: m.case_type == module.case_type, app.get_modules())
```

Comment:
This won't work for `AdvancedForms` but I'm not 100% clear on what the inclusion criteria are so this will probably do for now.